twilight-model/src/gateway/payload/incoming/message_poll_vote_remove.rs

```diff
@@ -0,0 +1,64 @@
+use crate::id::{
+    marker::{ChannelMarker, GuildMarker, MessageMarker, UserMarker},
+    Id,
+};
+use serde::{Deserialize, Serialize};
+
+/// Sent when a user removes a vote on a poll. If the poll allows multiple selection,
+/// one event will be sent per answer.
+#[derive(Clone, Debug, Deserialize, Eq, Hash, PartialEq, Serialize)]
+pub struct MessagePollVoteRemove {
+    /// ID of the answer.
+    pub answer_id: u64,
```

Comment:
Lets make this u8 to make it match with other places.
```suggestion
    pub answer_id: u8,
```